ts/packages/cli/src/commands/find.ts

```diff
@@ -0,0 +1,165 @@
+import { Flags } from '@oclif/core';
+import path from 'path';
+import { ConfigSchema } from '@configu/node';
+import { createInterface } from 'readline';
+import { createReadStream } from 'fs';
+import { once } from 'events';
+import _ from 'lodash';
+import { TMPL } from '@configu/ts';
+import FastGlob = require('fast-glob');
+import { BaseCommand } from '../base';
+
+export default class Find extends BaseCommand<typeof Find> {
+  static description = `Finding all config parameters and their usage in the source code based on CFGU files`;
+
+  static examples = [
+    {
+      description: `Find all config parameters and their usage in current directory`,
+      command: `<%= config.bin %> <%= command.id %>`,
+    },
+    {
+      description: `Find all config parameters and their usage in specific directory from provided CFGU file`,
+      command: `<%= config.bin %> <%= command.id %> --dir <PATH_TO_SCAN> --include <PATH_TO_CFGU>`,
+    },
+    {
+      description: `Find all unused config parameters in specific directory`,
+      command: `<%= config.bin %> <%= command.id %> --dir <PATH_TO_SCAN> --unused`,
+    },
+  ];
+
+  static flags = {
+    dir: Flags.directory({
+      description: 'Path to source code',
+      required: true,
+      default: '.',
+    }),
+    include: Flags.file({
+      description: 'Path to a specific .cfgu file (default: all CFGU files in the source code)',
+      required: false,
+      multiple: true,
+      default: [],
+    }),
+    exclude: Flags.file({
+      description: 'Glob pattern to ignore',
+      required: false,
+      multiple: true,
+      default: [],
+    }),
+    unused: Flags.boolean({
+      description: 'Only show unused parameters',
+      required: false,
+      default: false,
+    }),
+    templates: Flags.boolean({
+      description: '[default: false] Show parameters that are parts of templates and treat them as used parameters',
+      required: false,
+      default: false,
+      allowNo: true,
+    }),
+  };
+
+  private async updateIgnoredFiles(projectPath: string) {
+    const ignoredFiles = new Set(['build', 'dist', 'node_modules', ...this.flags.exclude]);
```

Comment:
Suggestion for the future, we might consider using https://www.npmjs.com/package/globby that has a built in support for gitignore files